### ISSUE TITLE
Categories are not synchronized with the Web client

### ISSUE THREAD

**Issue by XVilka** [mattermost/mattermost-mobile#5387]
#### Summary

If in Web-client Category is created it isn't shown in the mobile client.

### Environment Information
  - Device Name: any
  - OS Version: Android 10
  - Mattermost App Version: 1.42.1 (Build 3000354)
  - Mattermost Server Version: 5.34.0 (Build 5.34.1)

#### Steps to reproduce
1. Add a new category in Web client
2. Move some public  and private channels inside
3. Open the mobile app and see there are no categories

#### Expected behavior

Same categories as in Web

Also, there should be a way to create a new category and move in/out channels in the mobile app and then it should appear/change in the Web client.

#### Observed behavior (that appears unintentional)

No categories in mobile client

**Comment by marbaugh** [comment]
Agree, having this organization sync across web and mobile app would be very useful!

**Comment by marbaugh** [comment]
> The 1.47 aapp to be hitting the app stores today or tomorrow should have the caregories in sync.
> 
> Can you please report back once you tried the 1.47 release?
> 
> Thanks

Got the update last night and the syncing is great. Thanks!
